vector/src/main/java/im/vector/activity/RiotAppCompatActivity.kt

```diff
@@ -161,6 +161,11 @@ abstract class RiotAppCompatActivity : AppCompatActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onBackPressed() {
```

Comment:
You can not override "onBackPressed" for all the activities which inherit RiotAppCompatActivity